presto-main/src/main/java/com/facebook/presto/server/security/JsonWebTokenConfig.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.server.security;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigSecuritySensitive;
+
+import javax.validation.constraints.NotNull;
+
+public class JsonWebTokenConfig
+{
+    private String key;
+    private String requiredIssuer;
+    private String requiredAudience;
+
+    @NotNull
+    public String getKey()
+    {
+        return key;
+    }
+
+    @Config("http.authentication.jwt.key")
+    @ConfigSecuritySensitive
+    public JsonWebTokenConfig setKey(String key)
+    {
+        this.key = key;
+        return this;
+    }
+
+    public String getRequiredIssuer()
```

Comment:
Is null allowed?

Reply:
If you don't set is, all issuers are allowed.